recipes/enso_metrics/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "enso_metrics" %}
+{% set version = "v1.1.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/CLIVAR-PRP/ENSO_metrics/archive/refs/tags/{{ version }}.tar.gz
+    sha256: e933f373be1d793b801ed7155690fc4759a463da5931f18b5117ba9b259840b3    
+
+build:
+  number: 0
+  noarch: python
```

Comment:
Because the `scripts` argument is being used this cannot be `noarch`.

Ref:
* https://github.com/CLIVAR-PRP/ENSO_metrics/blob/6b5a3c33ec89a5c6de147bf6dc03872a89b9da90/setup.py#L68
* https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Noarch has been removed. Added skip for win and py2k.